openquake/server/apps.py

```diff
@@ -38,6 +38,16 @@ def ready(self):
             raise ValueError(
                 f'Invalid application mode: "{settings.APPLICATION_MODE}".'
                 f' It must be one of {settings.APPLICATION_MODES}')
+        if settings.APPLICATION_MODE == 'ARISTOTLE':
+            try:
+                # NOTE: optional dependency needed for ARISTOTLE
+                from timezonefinder import TimezoneFinder  # noqa
+            except ImportError:
+                raise ImportError(
+                    'The python package "timezonefinder" is not installed.'
+                    ' It is required in order to convert the UTC time to'
+                    ' the local time of the event. You can install it'
+                    ' running: pip install timezonefinder==6.5.2')
```

Comment:
I added the check here, raising the error if the webui is launched in ARISTOTLE mode and timezonefinder is not installed. I suppose the check in parsers.py can still remain, so the error will be raised also when running that code independently from the webui.